src/features/bookings/components/TicketBody/TicketBody.native.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { addDays, formatISO } from 'date-fns'
+import React from 'react'
+
+import { SubcategoryIdEnum, WithdrawalTypeEnum } from 'api/gen'
+import { bookingsSnap } from 'features/bookings/api/bookingsSnap'
+import { TicketBody } from 'features/bookings/components/TicketBody/TicketBody'
+import { render } from 'tests/utils'
+
+describe('TicketBody', () => {
+  const initialBooking = {
+    ...bookingsSnap.ongoing_bookings[0],
+    stock: {
+      ...bookingsSnap.ongoing_bookings[0].stock,
+      offer: {
+        ...bookingsSnap.ongoing_bookings[0].stock.offer,
+        isDigital: false,
+      },
+    },
+  }
+
+  describe('<QrCode/> display', () => {
+    it('should display the QR code when event subcategory is not in subcategories list without QR code display', () => {
+      const { queryByTestId } = render(<TicketBody booking={initialBooking} />)
+      expect(queryByTestId('qr-code')).toBeTruthy()
+    })
+
+    it('should not display the QR code when event subcategory is in subcategories list without QR code display', () => {
+      const booking = {
+        ...initialBooking,
+        stock: {
+          ...initialBooking.stock,
+          offer: {
+            ...initialBooking.stock.offer,
+            subcategoryId: SubcategoryIdEnum.FESTIVAL_MUSIQUE,
+          },
+        },
+      }
+
+      const { queryByTestId } = render(<TicketBody booking={booking} />)
+
+      expect(queryByTestId('qr-code')).toBeFalsy()
+    })
+  })
+
+  describe('Withdrawal', () => {
+    it('should not display on site withdrawal if withdrawal type is null', () => {
+      const booking = {
+        ...initialBooking,
+        stock: {
+          ...initialBooking.stock,
+          offer: {
+            ...initialBooking.stock.offer,
+            withdrawalType: null,
+          },
+        },
+      }
+      const { queryByTestId } = render(<TicketBody booking={booking} />)
+
+      expect(queryByTestId('withdrawal-info')).toBeFalsy()
+    })
+
+    describe('<NoTicket/> display', () => {
+      it('should display no ticket withdrawal wording and circled check icon', () => {
```

Comment:
```suggestion
      it('should display no ticket withdrawal wording', () => {
```

le test ne semble pas vérifier l'icon, ça semble etre un détail d'implem